.gitignore

```diff
@@ -17,4 +17,7 @@
 /scripts/automation/Radius/Config.ps1
 /scripts/automation/Radius/users.json
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+# Ignore PWSH CSV Import/Update Files:
```

Comment:
Remove commented out files eventually fix CSV tests to use just one placeholder file